microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/integration/config/MicroprofileConfigIntegrationTests.java

```diff
@@ -0,0 +1,309 @@
+     *                {@code mp.openapi.extensions.path} by Local Service Router deployments
+     * @tpPassCrit The server is logging expected {@code WFLYMPOAI0007} message because a deployment was configured
+     *             in order to have a custom url for {@code openapi} endpoint
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void testWarningIsLoggedBecauseOfNonConventionalOpenApiUrl() throws ConfigurationException, IOException {
+        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            ModelNodeLogChecker modelNodeLogChecker = new ModelNodeLogChecker(client, 100);
+            Assert.assertTrue(modelNodeLogChecker.logContains("WFLYMPOAI0007"));
+        }
+    }
+
+    /**
+     * @tpTestDetails Integration test to verify that missing MP config {@code mp.openapi.extensions.path} extension
+     *                property causes the server to log a warning about document generation being skipped for one given
+     *                deployment
+     * @tpPassCrit The server is logging expected {@code WFLYMPOAI0003} message because one deployment was not
+     *             properly configured in order to have a custom url for {@code openapi} endpoint
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void testWarningIsLoggedBecauseOfSkippedBadlyConfiguredDeployment() throws ConfigurationException, IOException {
+        try (final OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            ModelNodeLogChecker modelNodeLogChecker = new ModelNodeLogChecker(client, 100);
+            Assert.assertTrue(modelNodeLogChecker.logContains("WFLYMPOAI0003"));
+        }
+    }
+
+    /**
+     * @tpTestDetails Integration test to verify usage of MP Config core property {@code mp.openapi.servers} by
+     *                Local Service Router deployments
+     * @tpPassCrit The generated OpenAPI document {@code Server Records} contents are overridden by custom value
+     *             provided through {@code mp.openapi.servers} property
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void testServerRecordsAreOverriddenByConfigProperty() {
+        get(localServicesRouterTestOpenApiUrl)
+                .then()
+                .statusCode(200)
+                .contentType(equalToIgnoringCase("application/yaml"))
+                .body(containsString("url: https://xyz.io/v1"));
+    }
+
+    /**
+     * @tpTestDetails Integration test to verify usage of MP Config core property {@code mp.openapi.scan.disable} by
+     *                Local Service Router deployments
+     * @tpPassCrit The generated OpenAPI document {@code paths} property contents are not empty - i.e. the main
+     *             deployment related OpenAPI document is generated properly
+     *
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Ignore
```

Comment:
Add reason to `@Ignore` - e.g. link to the WFWIP issue

Reply:
Done.